src/NServiceBus.AcceptanceTests/Routing/When_using_instance_ids.cs

```diff
@@ -37,18 +35,21 @@ public class UnawareSender : EndpointConfigurationBuilder
         {
             public UnawareSender()
             {
-                EndpointSetup<DefaultServer>(c => { c.Routing().RouteToEndpoint(typeof(MyMessage), GetReceiverEndpoint()); });
+                EndpointSetup<DefaultServer>((c, r) =>
+                {
+                    c.UseTransport(r.GetTransportType()).Routing().RouteToEndpoint(typeof(MyMessage), ReceiverEndpoint);
+                });
             }
         }
 
         public class AwareSender : EndpointConfigurationBuilder
         {
             public AwareSender()
             {
-                EndpointSetup<DefaultServer>(c =>
+                EndpointSetup<DefaultServer>((c, r) =>
                 {
-                    c.Routing().RouteToEndpoint(typeof(MyMessage), GetReceiverEndpoint());
-                    c.Routing().Mapping.Physical.Add(new EndpointInstance(GetReceiverEndpoint(), "XYZ"));
+                    c.UseTransport(r.GetTransportType()).Routing().RouteToEndpoint(typeof(MyMessage), ReceiverEndpoint);
```

Comment:
we should probably provide an API in the acceptance testing for something like this, similar to the current AddMapping API which uses the legacy routing config.